targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_WIO_3G/PinNames.h

```diff
@@ -0,0 +1,268 @@
+    PF_7  = 0x57,
+    PF_8  = 0x58,
+    PF_9  = 0x59,
+    PF_10 = 0x5A,
+    PF_11 = 0x5B,
+    PF_12 = 0x5C,
+    PF_13 = 0x5D,
+    PF_14 = 0x5E,
+    PF_15 = 0x5F,
+
+    PG_0  = 0x60,
+    PG_1  = 0x61,
+    PG_2  = 0x62,
+    PG_3  = 0x63,
+    PG_4  = 0x64,
+    PG_5  = 0x65,
+    PG_6  = 0x66,
+    PG_7  = 0x67,
+    PG_8  = 0x68,
+    PG_9  = 0x69,
+    PG_10 = 0x6A,
+    PG_11 = 0x6B,
+    PG_12 = 0x6C,
+    PG_13 = 0x6D,
+    PG_14 = 0x6E,
+    PG_15 = 0x6F,
+
+    PH_0  = 0x70,
+    PH_1  = 0x71,
+
+    // ADC internal channels
+    ADC_TEMP = 0xF0,
+    ADC_VREF = 0xF1,
+    ADC_VBAT = 0xF2,
+
+    // Grove connector namings
+    D38         = PC_6,
+    D39         = PC_7,
+    A4          = PA_4,
+    A5          = PA_5,
+    A6          = PA_6,
+    A7          = PA_7,
+    TXD         = PB_6,
+    RXD         = PB_7,
+    I2C_SCL     = PB_8,
+    I2C_SDA     = PB_9,
+
+    // STDIO for console print
+#ifdef MBED_CONF_TARGET_STDIO_UART_TX
+    STDIO_UART_TX = MBED_CONF_TARGET_STDIO_UART_TX,
+#else
+    STDIO_UART_TX = PD_8,
+#endif
+#ifdef MBED_CONF_TARGET_STDIO_UART_RX
+    STDIO_UART_RX = MBED_CONF_TARGET_STDIO_UART_RX,
+#else
+    STDIO_UART_RX = PD_9,
+#endif
+
+
+    // Quectel UG96 cellular modem
+    MDMTXD  = PA_2,
+    MDMRXD  = PA_3,
+    MDMCTS  = PA_0,
+    MDMRTS  = PA_1,
+
+    MDMSTAT = PB_15,
+    MDMDTR  = PC_5,
+    M_POWR  = PE_9,
+
+/*
+    MDMDCD = NC, // Data Carrier Detect
+    MDMDSR = NC, // Data Set Ready
+    MDMRI  = NC, // Ring Indicator
+*/
+
+    // Internal control signals
+    RGB_POWR     = PE_8,
+    GRO_POWR     = PB_10,
+    PWRKEY       = PB_5,
+    RESET_MODULE = PD_5,
+
+    // Generic signals namings
+    LED1        = PB_11,  // LD1 = GREEN
+    LED2        = PB_12,  // Blue
+    LED3        = PB_13, // Red
+    LED4        = PB_14,
+    LED_RED     = LED2,
+    USER_BUTTON = PC_13,
+    // Standardized button names
+//    BUTTON1 = USER_BUTTON,
+    SERIAL_TX   = STDIO_UART_TX, // Virtual Com Port
+    SERIAL_RX   = STDIO_UART_RX, // Virtual Com Port
+    USBTX       = STDIO_UART_TX, // Virtual Com Port
+    USBRX       = STDIO_UART_RX, // Virtual Com Port
+    SPI_MOSI    = PC_12,
+    SPI_MISO    = PC_11,
+    SPI_SCK     = PC_10,
+    SPI_CS      = PD_0,
+//    PWM_OUT     = D9,
+
+/*
+    //USB pins
+    USB_OTG_HS_ULPI_D0 = PA_3,
+    USB_OTG_HS_SOF = PA_4,
+    USB_OTG_HS_ULPI_CK = PA_5,
+    USB_OTG_FS_SOF = PA_8,
+    USB_OTG_FS_VBUS = PA_9,
+    USB_OTG_FS_ID = PA_10,
+    USB_OTG_FS_DM = PA_11,
+    USB_OTG_FS_DP = PA_12,
+    USB_OTG_HS_ULPI_D1 = PB_0,
+    USB_OTG_HS_ULPI_D2 = PB_1,
+    USB_OTG_HS_ULPI_D7 = PB_5,
+    USB_OTG_HS_ULPI_D3 = PB_10,
+    USB_OTG_HS_ULPI_D4 = PB_11,
+    USB_OTG_HS_ID = PB_12,
+    USB_OTG_HS_ULPI_D5 = PB_12,
+    USB_OTG_HS_ULPI_D6 = PB_13,
+    USB_OTG_HS_VBUS = PB_13,
+    USB_OTG_HS_DM = PB_14,
+    USB_OTG_HS_DP = PB_15,
+    USB_OTG_HS_ULPI_STP = PC_0,
+    USB_OTG_HS_ULPI_DIR = PC_2,
+    USB_OTG_HS_ULPI_NXT = PC_3,
+*/
+
```

Comment:
I think all of these un-used USB pins can be removed..

Reply:
Done